tests/integration/render.rs

```diff
@@ -56,11 +143,97 @@ fn render_config_example() -> anyhow::Result<()> {
         include_str!("../../tests/data/rendered_manifests/edge-eu-w4/my-app/manifest.yaml")
     );
 
-    // todo extend assertions here while changing the chart under test
-    // todo this test could also benefit from some utility functions/macros to make it less verbose
+    // // todo extend assertions here while changing the chart under test
+    // // todo this test could also benefit from some utility functions/macros to make it less verbose
+
+    // delete temp folder
+    drop_temp_folders(&format!("{}", config.parent().unwrap().to_string_lossy()))?;
+
+    Ok(())
+}
+
+#[test]
+fn pipe_output_to_a_tool_that_exists() -> anyhow::Result<()> {
+    let mut cmd = Command::cargo_bin(BIN_NAME)?;
+
+    let config = generate_config().unwrap();
+
+    cmd.arg("render").arg("--pipe=grep 'image'").arg(&config);
+
+    cmd.assert().success();
+
+    let edge_rendered_output = format!(
+        "{}/manifests/edge-eu-w4/my-app/manifest.yaml",
+        config.parent().unwrap().to_string_lossy()
+    );
+
+    let mut yaml = std::fs::File::open(edge_rendered_output).unwrap();
+
+    let mut contents = "".to_string();
+    yaml.read_to_string(&mut contents)?;
+
+    // assert that the output contains only images related content
+    assert_eq!(
+        contents.contains(
```

Comment:
I'd prefer an exact match in this case. You can store the expected string literal in a variable and then assert it. Or you can store it in a file and load it using `include_str!` like `render_config_example`. But for a 2 line file, that seems overkill.